WordPress/Classes/ViewRelated/Post/PrepublishingViewController.swift

```diff
@@ -31,10 +31,42 @@ class PrepublishingViewController: UITableViewController {
 
     private let completion: (AbstractPost) -> ()
 
+    private func rowTitle(rowID: PrepublishingIdentifier) ->  NSAttributedString {
```

Comment:
Instead of using a `NSAttributedString` to style the color/fontsize, what do you think about subclassing `WPTableViewCell` and making a custom cell?

I also wonder how the cells would look if we put the description label right below the title label.